### ISSUE TITLE
Admin General Settings Panel - Ensure buttons have discernible text

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#239]
WCAG 4.1.2: Ensures buttons have discernible text (.o_dropdown_button[draggable="false"][role="button"])
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Ensures buttons have discernible text (button-name - https://dequeuniversity.com/rules/axe/3.5/button-name?application=msftAI)

Target application: Odoo - http://localhost:8069/web#action=142&cids=1&id=&menu_id=95&model=res.config.settings&view_type=form

Element path: div[name="paperformat_id"] > .o_input_dropdown > .o_dropdown_button[draggable="false"][role="button"]

Snippet: `<a role="button" class="o_dropdown_button" draggable="false"></a>`

How to fix: 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element's default semantics were not overridden with role="presentation"
  Element's default semantics were not overridden with role="none"
  Element has no title attribute or the title attribute is empty

Environment: Chrome version 80.0.3987.163

====

WCAG 4.1.2: Ensures buttons have discernible text (.o_dropdown_button[draggable="false"][role="button"])
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Ensures buttons have discernible text (button-name - https://dequeuniversity.com/rules/axe/3.5/button-name?application=msftAI)

Target application: Odoo - http://localhost:8069/web#action=142&cids=1&id=&menu_id=95&model=res.config.settings&view_type=form

Element path: div[name="external_report_layout_id"] > .o_input_dropdown > .o_dropdown_button[draggable="false"][role="button"]

Snippet: `<a role="button" class="o_dropdown_button" draggable="false"></a>`

How to fix: 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element's default semantics were not overridden with role="presentation"
  Element's default semantics were not overridden with role="none"
  Element has no title attribute or the title attribute is empty

Environment: Chrome version 80.0.3987.163

====

These accessibility issues were found using Accessibility Insights for Web 2.16.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
